### ISSUE TITLE
Color Configuration for Social Cards

### ISSUE THREAD

**Issue by grantmcd** [squidfunk/mkdocs-material#2876]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I want to suggest an idea and checked that ...

- [X] ... to my best knowledge, my idea wouldn't break something for other users
- [X] ... the documentation does not mention anything about my idea
- [X] ... there are no open or closed issues that are related to my idea

### Description

We just enabled the new social card plugin on our docs site. However we noticed that the card does not use the colors in our custom color schemes.

It would be great if we could configure the colors of the social cards.

### Use Cases

To configure the look of the social cards so that it matches the rest of your site.

**Comment by squidfunk** [comment]
Thanks for suggesting! I've plans already for adding the possibility to explicitly define the background and foreground color through the configuration of the plugin in the coming days/weeks! Additionally, I will add support for background images.

**Comment by squidfunk** [comment]
The latest Insiders commit adds support for custom colors.

_Example_:

```yaml
site_name: Pied Piper
site_description: Compression. Data. The Internet.
theme:
  font:
    text: Titillium Web
  icon:
    logo: fontawesome/brands/pied-piper-hat
plugins:
  - social:
      cards_color:
        fill: "#FFF"
        text: "#36837f"
```

_Result_:

![getting-started](https://user-images.githubusercontent.com/932156/128597807-56f23da1-f2fc-4030-b39a-93b0eb6043a3.png)

`fill` and `text` can be set to any HEX color value or CSS color keyword.

I'll also add this to the documentation in a few minutes.

**Comment by squidfunk** [comment]
Released as 7.2.2-insiders-2.13.0.

**Comment by grantmcd** [comment]
Beautiful! Thank you!
